dockerfiles/Dockerfile.oracle

```diff
@@ -0,0 +1,68 @@
+FROM ghcr.io/verrazzano/verrazzano-base:v1.0.0-20230327155846-4653b27 as builder-base
```

Comment:
It is better if the automation can supply the latest image as an argument, so that the image is built with the latest base image.

Reply:
Good idea, I can do it as a follow up.